### PR TITLE
[Doc] Emily/doc update

### PR DESCRIPTION
**Pull request by chuneuny-emily**
This PR removes the duplicated and cyclical category context.

### Checklist

- [ ] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md)
- [x] I have updated the [documentation](
  https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have [linked related issues](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,11 +18,9 @@ CVAT annotations                             ---> Publication, statistics etc.
 <!--lint enable fenced-code-flag-->
 
 - [Getting started](https://openvinotoolkit.github.io/datumaro/docs/getting_started)
-- [Examples](https://openvinotoolkit.github.io/datumaro/docs/getting_started/#examples)
 - [Features](#features)
 - [User manual](https://openvinotoolkit.github.io/datumaro/docs/user-manual)
 - [Developer manual](https://openvinotoolkit.github.io/datumaro/api)
-- [Python API](https://openvinotoolkit.github.io/datumaro/docs/python-api)
```
Comment (vinnamkim) [inline README.md:25]: Why do we have to drop all the notebook example docs?

File: site/content/en/docs/getting_started.md [hunk 2]
```diff
@@ -15,21 +15,68 @@ To read about the design concept and features of Datumaro, go to the [design sec
...
 ```
+``` bash
+# From the GitHub repository:
+pip install 'git+https://github.com/openvinotoolkit/datumaro[default]'
+```
+
+Read more about choosing between `datumaro` and `datumaro[default]`
+[here](#core-install).
+
+<a id="installation-plugins"></a>
```
Comment (vinnamkim) [inline site/content/en/docs/getting_started.md]: Is it possible to remove this line?

File: site/content/en/docs/getting_started.md [hunk 3]
```diff
@@ -15,21 +15,68 @@ To read about the design concept and features of Datumaro, go to the [design sec
...
+# From PyPI:
 pip install datumaro[default]
 ```
+``` bash
+# From the GitHub repository:
+pip install 'git+https://github.com/openvinotoolkit/datumaro[default]'
+```
+
+Read more about choosing between `datumaro` and `datumaro[default]`
+[here](#core-install).
```
Comment (vinnamkim) [inline site/content/en/docs/getting_started.md]: Is it possible to link by `[here](#customizing-installation)` without introducing `<a id="core-install"></a>`?

File: .github/workflows/github_pages.yml [hunk 4]
```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - develop
+      - releases/v1.0.0
```
Comment (yunchu) [inline .github/workflows/github_pages.yml:6]: please remove "develop" target branch from the list.

File: README.md (hunk 1 above)
Comment (chuneuny-emily) [reply README.md:25]: Documentation website including Python API will be opened via user manual link.

File: .github/workflows/github_pages.yml (hunk 4 above)
Comment (chuneuny-emily) [reply .github/workflows/github_pages.yml:6]: I see.

**Review by vinnamkim — APPROVED** [review]
LGTM.

**Review by yunchu — APPROVED** [review]

**Pull request merged**